dlls/client.cpp

```diff
@@ -1194,7 +1194,7 @@ void StartFrame( void )
 
 				// have to delay here or engine gives "Tried to write to
 				// uninitialized sizebuf_t" error and crashes...
-				pause_time = gpGlobals->time + 1;
```

Comment:
Currently, you changed really nothing exclude how compiler optimizes operations with digit literal promotion:
old variant will be equal to:
```
pause_time = gpGlobals->time + 1.0f;
```
but new variant will be equal to:
```
pause_time = (float)((double)gpGlobals->time + 1.0);
```

Reply:
You can exclude this commit via `git rebase`.

Reply:
Anyway I recommend to use integer or float literals(.0f) if type of used variables is float.
For differrent types in C and C++ works promotion rules and compiler can't just replace float and double literals.
Double literals(.0) eats more memory and double-to-float/float-to double casts is not cheap for CPU.